content/en/core/releases/3.15.0.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "3.15.0 release notes"
+linkTitle: "3.15.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+3.15.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+None.
+
+## Highlights
+
+### Support reporting of Bikram Sambat dates using text forms
+
+SMS forms can now include a date form for submitting an exact date in Bikram Sambat format. Learn how to use this feature in [the documentation](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/forms/#app_settingsjson-forms).
+
```

Comment:
```suggestion
[#4613](https://github.com/medic/cht-core/issues/4613): Support reporting of dates as exact date using text forms
```

Minor, but in previous release docs, we have included a link to each of the the issues in this "Highlights" section.

Reply:
Yeah I hummed and harred about that. I'll add it.